scripts/create_perf_json.py

```diff
@@ -335,9 +335,20 @@ def get(key: str) -> str:
                 self.unit = unit_fixups[self.unit]
             elif self.unit == "NCU" and self.event_name == "UNC_CLOCK.SOCKET":
                 self.unit = "CLOCK"
-            elif self.unit == "PCU" and self.umask:
-                # TODO: convert to right filter for occupancy
-                self.umask = None
+            elif self.event_name.startswith("UNC_P_POWER_STATE_OCCUPANCY"):
```

Comment:
Notes:
* Reviewed uncore performance monitoring document https://www.intel.com/content/www/us/en/develop/download/intel-xeon-processor-e5-v2-and-e7-v2-product-families-uncore-performance-monitoring.html as an example (IVB).
* See table 2-121.